src/components/mdIcon/mdIcon.vue

```diff
@@ -3,6 +3,8 @@
 
   <md-image class="md-icon" :class="[themeClass]" :md-src="imageSrc" v-else-if="imageSrc"></md-image>
 
+  <i class="md-icon md-icon-font" aria-hidden="true" :class="[themeClass]" v-else-if="mdIconset"></i>
```

Comment:
Then here you don't need a else-if because your iconset will evaluate as a class:
``` vue
<i class="md-icon" :class="[themeClass, mdIconSet]" :aria-hidden="!!mdIconSet" v-else>
  <slot></slot>
</i>
```